### DISCUSSION TITLE
category and keyword list is empty

### DISCUSSION THREAD

**Discussion by kuduacz** [nextcloud/cookbook#1698] [FAQ and user questions]
How to create a category and keyword's?
Cant choose anything when creating new recipe

**Comment by christianlupus** [comment]
Currently, you can create categories just on the fly. You can (during editing a recipe) click in the list of categories and either select an existing one (which you do not have yet) or just *type* the name of a new one. After saving, the category will just appear. You do not need to explicitly create a category.

The keywords are similar but you can have multiple per recipe.


https://github.com/nextcloud/cookbook/assets/8202922/fe31c12a-4499-4fe1-8c84-0658ade94132
